src/store/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+
+import virtualStudies from './virtualStudies';
+
+export default combineReducers({
+  virtualStudies,
```

Comment:
@jberube I would add a `sqon` reducer as well as this can be re-used across several "pages".

Reply:
Yes, when time comes.